aws-athena/views/default-vw_pin_value.sql

```diff
@@ -11,6 +11,13 @@ WITH stage_values AS (
         parid AS pin,
         taxyr AS year,
         -- Mailed values
+        MAX(
```

Comment:
**nitpick:** You could probably replace `MAX()` with `ARBITRARY()` for some potential speed gains. But I would test to see if `ARBITRARY()` can return `NULL` first.